### DISCUSSION TITLE
Can I disable file dragging to the editor?

### DISCUSSION THREAD

**Discussion by dav-sap** [GrapesJS/grapesjs#3849] [Q&A] [answered]
I have an issue where you can just drag files inside the editor and drop them and they are not interpeted to a valid component and then it just get complicated from there on.
Is there an option to disable this file drag&drop to the editor?
Specifically with image files.

**Comment by artf** [comment, marked as answer]
You can leverage `canvas:dragdata` and decide what to drop and what to skip https://jsfiddle.net/artur_arseniev/87rcb24n/ but there is no way to disable completely the D&D as you wouldn't be able to move components or drop blocks.

**Comment by dav-sap** [comment]
Thanks @artf found my solution with this fiddle example.
